.github/workflows/dispatch-release-note.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
           curl \
           -X POST \
           -H "Accept: application/vnd.github+json" \
-          -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+          -H "Authorization: Bearer ${{ secrets.PAT }}" \
```

Comment:
How about using GitHub App?

Reply:
Good idea! Github Apps is more convinient for organization secrets rather than presonal secrets.
Let me try Github Apps.